query/aggregate/aggregate.go

```diff
@@ -74,7 +74,7 @@ func groupBy(list api.SeriesList, tags []string) []group {
 	return result
 }
 
-var aggregateMap = map[string]func([]float64) float64{
+var AggregateMap = map[string]func([]float64) float64{
```

Comment:
i'm wondering whether this is even needed - because we don't need a map to store it right? Technically we can just have:

```
func Sum([]float64) float64 {}
...
func Mean([]float64) float64 {}
...
right?
```